plugin/externalPluginEntry.go

```diff
@@ -229,8 +229,13 @@ func (e *externalPluginEntry) Stream(ctx context.Context) (io.ReadCloser, error)
 }
 
 func (e *externalPluginEntry) Exec(ctx context.Context, cmd string, args []string, opts ExecOptions) (ExecCommand, error) {
+	// Serialize opts to JSON
+	optsJSON, err := json.Marshal(opts)
```

Comment:
Do we really want to pass `stdin`? Seems like we only want a subset of options.

Reply:
Oh, looks like you dealt with that with some json markup.

Reply:
`json:"-"` in https://github.com/ekinanp/wash/blob/update-ep-exec/plugin/types.go#L69 tells Go not to include that field in the serialization, so it won't be included. I'll add a comment about that though.